### ISSUE TITLE
Buttons do not work in "Do you want to show a rich preview?" message

### ISSUE THREAD

**Issue by sbleon** [integrations/slack#948]
![rich_preview_question](https://user-images.githubusercontent.com/198711/66067594-e9be0b80-e519-11e9-86dd-7a16f5f653ca.png)

When I post a GH URL in a subscribed channel, the message above pops up. However, neither of the buttons works. When I click either button, they both flash briefly, as if being momentarily disabled. If I'm using Slack in my browser, I can see the following in the console:

```
Oct-2 13:41:11.606 [API-Q] (T03MW33L4) afd3ad0f-1570038071.605 chat.attachmentAction is ENQUEUED gantry-shared.59dd12e.min.js?cacheKey=gantry-1570034160:1
Oct-2 13:41:11.608 [API-Q] (T03MW33L4) afd3ad0f-1570038071.605 chat.attachmentAction is ACTIVE gantry-shared.59dd12e.min.js?cacheKey=gantry-1570034160:1
Oct-2 13:41:11.746 [API-Q] (T03MW33L4) afd3ad0f-1570038071.605 chat.attachmentAction is REJECTED: API not ok user_not_visible backend.js:6 
```

I've seen a recommendation to use `/github signin` to integrate my Slack profile with Github, but that doesn't work either. Slackbot responds with: `/github failed with the error "user_not_visible"`

**Comment by stuartsierra** [comment]
I have observed the same issue when using the Slack app.

Slack Version 4.1.0 (Production)
macOS Mojave Version 10.14.5

**Comment by rossisdead** [comment]
I'm having a slightly different issue related to this prompt. I can hit "Dismiss" and the prompt disappears, but the next time I post a Github url it will prompt me again.

**Comment by ElijahLynn** [comment]
I have same issue in a private repo and get that same error `/github failed with the error "user_not_visible"`

**Comment by ElijahLynn** [comment]
Related > https://github.com/integrations/slack/issues/954

**Comment by ElijahLynn** [comment]
This appears to be because we may be restricted Slack single or multi-channel guests accounts), which I am. Can you all confirm your account types?

https://github.com/slackhq/slack-api-docs/blob/ba81cad6bdd46a656a7b5725b49a64cf32730316/methods/im.open.json#L14

**Comment by sbleon** [comment]
I believe mine is a multi-channel guest account.

**Comment by stuartsierra** [comment]
Yes, I observed this in Slack accounts where I was a multi-channel guest.
